content/webapp/components/EventsByMonth/EventsByMonth.tsx

```diff
@@ -45,9 +45,9 @@ const EventsByMonth: FunctionComponent<Props> = ({ events, links }) => {
   return (
     <>
       <Space $v={{ size: 'm', properties: ['margin-bottom'] }}>
-        <CssGridContainer>
-          <div className="css-grid">
-            <div className={cssGrid(gridSize12())}>
+        <Container>
+          <div className="grid">
+            <GridCell $sizeMap={gridSize12()}>
```

Comment:
Does it always have to be wrapped in a div with a class of `grid`? Could we add that to the component instead (maybe optional?) and remove the class?